app/src/main/java/com/example/recipeapp/adapters/InventoryAdapter.java

```diff
@@ -66,6 +70,52 @@ public void bind(Ingredient ingredient) {
             binding.tvName.setText(ingredient.getName());
             binding.tvCount.setText(String.valueOf(ingredient.getCount()));
             binding.tvUnit.setText(ingredient.getUnit());
+            binding.ibAdd.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    add(ingredient);
```

Comment:
nice job encapsulating all of the `add` (and `remove`) logic into its own method! 🎉